tests/test_wave_01.py

```diff
@@ -1,10 +1,11 @@
+#from turtle import update
```

Comment:
An extraneous/accidental auto-import from VS Code? I recommend turning that feature off in the VS Code settings. No need to commit this commented out line, it's better to just delete it.